code/datums/mood_events/generic_negative_events.dm

```diff
@@ -361,3 +361,12 @@
 	description = "<span class='boldwarning'>I hate being sprayed with water!</span>\n"
 	mood_change = -1
 	timeout = 30 SECONDS
+
+/datum/mood_event/gamer_withdrawal
+	description = span_warning("I wish I was gaming right now...\n")
+	mood_change = -5
+
+/datum/mood_event/gamer_lost
+	description = span_boldwarning("I hate losing after a heated gamer moment!\n")
+	mood_change = -10
+	timeout = 10 MINUTES
```

Comment:
if you are angry over a game for a whole 10 minutes after losing then I would honestly be impressed.

Reply:
Gamers never forget.